pilot/cmd/pilot-agent/main.go

```diff
@@ -66,6 +66,8 @@ var (
 	concurrency              int
 	templateFile             string
 	disableInternalTelemetry bool
+	appReadyURL              string
+	appLiveURL               string
```

Comment:
can we rename these as kubernetes style livenessProbeURL and readinessProbeURL

Reply:
Done.